docs/_config.yml

```diff
@@ -0,0 +1,98 @@
+# © 2020 and later: Unicode, Inc. and others.
+# License & terms of use: http://www.unicode.org/copyright.html
+
+title: ICU Documentation
+email: icu-support@icu-project.org
+description: >- # this means to ignore newlines until "baseurl:"
+  ICU is a mature, widely used set of C/C++ and Java libraries providing Unicode
+  and Globalization support for software applications. The ICU User Guide provides
+  documentation on how to use ICU.
+
+baseurl: "/icu" # the subpath for the site
+url: "https://unicode-org.github.io" # the base hostname & protocol for your site
+
+twitter_username: unicode
+github_username:  unicode-org
+
+remote_theme: pmarsceill/just-the-docs
+
+# GitHub uses it's own markdown renderer called CommonMarkGhPages.
+#
+# To use this we'd need to set the following options:
+#     markdown: CommonMarkGhPages
+#     Plugins:
+#       jekyll-commonmark-ghpages
+#  
+#     commonmark:
+#       options: ["SMART", "FOOTNOTES"]
+#       extensions: ["strikethrough", "autolink", "table", "tagfilter"]
+#
+# However, it doesn't work well with Jekyll and the Just-The-Docs theme,
+# and causes too many formatting problems. Additionally, it also doesn't
+# have any support for auto-generating a Table of Contents (TOC) section
+# for a page. Due to these reasons, we instead use the recommended
+# markdown renderer for Jekyll, called kramdown.
+#
+markdown: kramdown
+
+# Notes on plugins:
+#  - GitHub Pages ignores all plugins included in the Gemfile.
+#    - Only the ones listed in the _config.yml file (this file) are used.
+#  - There is a very limited set of plugins that GH Pages supports (for security
+#    reasons). The listing is here: https://pages.github.com/versions
+#  - The jekyll-sitemap plugin generates a "sitemap.xml" for search engines.
+#  - We need the "jemoji" plugin for rendering emoji like ":point_right:".
+plugins:
+  - jekyll-remote-theme
+  - jekyll-sitemap
+  - jemoji
+
+search_enabled: true
+
+# Color scheme currently only supports "dark" or nil (light).
+color_scheme: nil
+
+# Aux links for the upper right navigation
+aux_links:
+  "API Docs":
+    - "//unicode-org.github.io/icu-docs"
+  "ICU on GitHub":
+    - "//github.com/unicode-org/icu"
+  "ICU Home Page":
+    - "http://site.icu-project.org/"
+  "Unicode Home Page":
+    - "//www.unicode.org"
```

Comment:
Out of curiosity, where does the `//...` notation for a URL come from vs. the `http[s]` in other links?

Reply:
Good question! 
The `//...` notation for a URL effectively means "use the same protocol" for the link. So if you're on a `https` page it will use that, if you're on a `http` page it will use that. (I believe this is called a "protocol-relative" URL).

I thought it might be better to use the relative protocol links for the GitHub pages (since it's going from a GitHub hosted URL to another GitHub hosted URL).

I had to use the HTTP explicit protocol for the ICU Home Page though, as that unfortunately doesn't support HTTPS.

The Unicode site supports both, so we could use either. I used the `//` here, but maybe using `https` explicitly would be better?

Reply:
Clever, and good to know, but for domains where we know that https works (GitHub & unicode.org) I would just specify that directly.

Reply:
Thanks! I will force-push an update to change the URLs to be explicit.